goci-interfaces/goci-ui/src/main/java/uk/ac/ebi/spot/goci/ui/controller/SolrSearchController.java

```diff
@@ -74,17 +74,17 @@ public void doSolrSearch(
         else {
             addRowsAndPage(solrSearchBuilder, maxResults, page);
         }
-        addQueryFilter(solrSearchBuilder);
+        // addQueryFilter(solrSearchBuilder);
         addQuery(solrSearchBuilder, query);
 
         // dispatch search
         dispatchSearch(solrSearchBuilder.toString(), response.getOutputStream());
     }
 
     // Use queryFilter to return match to term synonym higher in the result list
-    private void addQueryFilter(StringBuilder solrSearchBuilder) {
-        solrSearchBuilder.append("&defType=dismax&qf=title%5E2.0+synonyms%5E20.0+parent%5E2.0+text%5E1.0");
-    }
+    //private void addQueryFilter(StringBuilder solrSearchBuilder) {
+    //    solrSearchBuilder.append("&defType=dismax&qf=title%5E2.0+synonyms%5E20.0+parent%5E2.0+text%5E1.0");
+    //}
 
```

Comment:
Because the changes in [GOCI-2476_TW-rank-synonym-results-higher](https://github.com/EBISPOT/gwas-ui/tree/(GOCI-2476_TW-rank-synonym-results-higher)  break the variant page. I couldn't test the variant page without disabling these statement. I had to commit these changes otherwise it wouldn't be deployed to the test server.